### PR TITLE
Contributing guidelines

### PR DESCRIPTION
**Pull request by zrzka**
Ongoing work for #447. @omz, check it please. Waiting for ack to merge it or you can merge it directly.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+* Pythonista 3.1.1 (311015), Default interpreter 3.6.1
+* iOS 11.0.3, model iPad, resolution (portrait) 2048.0 x 2732.0 @ 2.0
+```
+
+If you don't know from where to get this information, download and run [sysinfo.py][pythonista-sysinfo-script] script.
+
+Why? Some bugs may appear on specific device, specific iOS version and it's very helpful to know more about your system. Problem can be identified and fixed more quickly.
+
+If you feel bad about sharing system information of your device, don't attach it. It's mandatory, but it's really helpful.
```
Comment (dgelessus) [inline CONTRIBUTING.md]: That should probably be "It's *not* mandatory".

Comment (zrzka) [reply CONTRIBUTING.md]: Fixed, thanks for catching this!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+#### System information
+
+Here's an example of system information to attach (preformatted Markdown):
+
+```
+**System Information**
+
+* Pythonista 3.1.1 (311015), Default interpreter 3.6.1
+* iOS 11.0.3, model iPad, resolution (portrait) 2048.0 x 2732.0 @ 2.0
```
Comment (cclauss) [inline CONTRIBUTING.md]: I would think that __model iPad5,4__ would be more helpful than __model iPad__.

Comment (zrzka) [reply CONTRIBUTING.md]: Fixed, `platform.machine()` used instead of `device.model()`.

**Pull request merged**